pkg/labels/labels_test.go

```diff
@@ -485,3 +485,82 @@ func TestLabels_Has(t *testing.T) {
 		testutil.Equals(t, test.expected, got, "unexpected comparison result for test case %d", i)
 	}
 }
+
+func TestLabels_Get(t *testing.T) {
+	tests := []struct {
```

Comment:
This table test for single line test might be an overkill..

The same test will have just 2 lines...

Can we add just two lines instead?